Adyen/UI/Form/Items/FormImageItem/FormImageItem.swift

```diff
@@ -20,14 +20,17 @@ public class FormImageItem: FormItem, Hidable {
     public var identifier: String?
 
     /// The size of the Image.
-    public var size: CGSize?
+    public var size: CGSize = .init(width: 46, height: 46)
```

Comment:
It is probably not needed here, since it is assigned on `init`

Reply:
Fixed. Here is the commit: 32a3cdc9cc0d9e1ff71bdd9886176033095764dd